model/Deserialization/Classes/Deserialization.md

```diff
@@ -0,0 +1,36 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# Deserialization
+
+## Summary
+
+An element that can store information about a deserialization event, minted on the fly and not persisted in the payload.
+
+## Description
+
+tbd.
+
+Propreties should be used as follows:
+
+- element, to encode the content that was generated by when parsing the file
+- name, to optionally encode the file name, if it was a file
+- verifiedBy, to encode relevant hashes
+
+TODO:
+- probably could benefit from a better class name!
+- encode the format?
+- allow to save the content as ByteString?
+
+## Metadata
+
+- name: Deserialization
+- SubclassOf: /Core/Artifact
```

Comment:
I though about suggesting a subclass of File, but after thinking about it we would want to cover the case where something is serialized as a network stream.